src/components/CardAboutShelter/components/InfoRow/InfoRow.tsx

```diff
@@ -35,7 +35,17 @@ const InfoRow = React.forwardRef<HTMLDivElement, IInfoRowProps>(
           <span className={cn('font-normal', value ? 'text-nowrap' : '')}>
             {label}
           </span>
-          <span className="md:flex">{ValueComp}</span>
+          <span className="md:flex">
+          {ValueComp}
+          {
+              clipboardButton ?
```

Comment:
Poderia ser somente como abaixo não, já que é um boolean:
```suggestion
              clipboardButton &&
```

Reply:
Correto.